hw/ip/csrng/data/csrng.hjson

```diff
@@ -133,6 +133,13 @@
       name: "CSRNG.MODE.NONDETERMINISTIC",
       desc: "The CSRNG can be set up to generate non-deterministic random values by using a random seed from the entropy source."
     }
+    {
+      name: "CSRNG.LIFECYCLE.DEBUGENABLE",
+      desc: '''
+            The CSRNG has a debug enable signal, which allows testing with known entropy values.
+            For security reasons, this must be disabled as the life cycle of the chip goes to dev or prod.
+            '''
+    }
```

Comment:
Thanks for adding this @marnovandermaas . IIUC, this description doesn't match the actual implementation though. The only use of the `lc_hw_debug_en_i` is for seed diversification and this is what we should validate in SiVal, from `csrng_core.sv`:
```sv
  // SEC_CM: CONSTANTS.LC_GATED
  assign seed_diversification = lc_hw_debug_on_fo[0] ? RndCnstCsKeymgrDivNonProduction :
                                                       RndCnstCsKeymgrDivProduction;
```
Both constants are random and secret netlist constants. But as you noted, this will require changing the lifecycle state. For known-answer testing (which this description seems to target), we either test ENTROPY_SRC alone or CSRNG using `flag0` (ignores ENTROPY_SRC input).

There is however an input signal from OTP called `otp_en_csrng_sw_app_read_i` that we use to determine whether the internal state is readable or not. Based on [this internal discussion](https://github.com/lowRISC/opentitan-integrated/pull/319/files#r1303295142) it looks like this value will however always be set to true in practice.